packages/@react-aria/selection/src/useSelectableList.ts

```diff
@@ -18,21 +18,57 @@ import {useCollator} from '@react-aria/i18n';
 import {useSelectableCollection} from './useSelectableCollection';
 
 interface SelectableListOptions {
+  /**
+   * An interface for reading and updating multiple selection state.
```

Comment:
Aren't these also in one of @LFDanLu PRs? Can someone compare and pick the best?

Reply:
Nah I think I did useSelectableCollection but it is prob similar for some of the type descriptions

Reply:
I believe I copied it from Daniel :)